pkg/kubelet/rkt/rkt.go

```diff
@@ -830,6 +830,13 @@ func (r *Runtime) generateRunCommand(pod *api.Pod, uuid string) (string, error)
 
 	var hostname string
 	var err error
+
+	// Overlay fs is not supported for SELinux yet.
```

Comment:
If I remember correctly, CoreOS carries a patch such that selinux does work with overlayfs in that environment. See https://coreos.com/releases/#808.0.0 and https://github.com/coreos/coreos-overlay/pull/1703

It would be nice to have this at least work in that case.
My immediate thought is we could try it and see if it works, and go with that. There might be a less hacky way to do this though. I'd at least like a bit more thought/discussion.

Perhaps @mjg59 has a good idea for what to do here?